src/components/TokenIcon/index.stories.tsx

```diff
@@ -57,4 +57,4 @@ export const Default = () => (
   </>
 );
 
-export const ShowSymbol = () => <TokenIcon token={TOKENS.usdc} showSymbol />;
+export const WithShowSymbol = () => <TokenIcon token={TOKENS.usdc} showSymbol />;
```

Comment:
nit: with makes me think it is a story decorator and not a story itself

Reply:
Oops yes sorry, this change got lost when I closed the other PR, I should have rebased from it.